store/src/java/com/zimbra/cs/service/account/GetInfo.java

```diff
@@ -254,6 +261,16 @@ public Element handle(Element request, Map<String, Object> context) throws Servi
         for (GetInfoExt extension : extensions) {
             extension.handle(zsc, response);
         }
+
+        // we do not have any ldap attrs to define whether powerpaste service is installed and running
+        // so check if powerpaste service is installed and running by checking the installed directory and connectivity
+        // if yes return powerpasteEnabled = true else return powerpasteEnabled = false
+        if (checkIfPowerpasteInstalled()) {
```

Comment:
Can we write the following block in a single line?
like : `response.addAttribute("powerpasteEnabled", checkIfPowerpasteInstalled());`

Reply:
can be done